### PR TITLE
fix: add loader for gif

### LINKED ISSUE #1878: Primeng/Primeui webpack Module parse failed Unexpected character
Error on adding primeng/primeui to a angular-cli webpack project

1. OS? 
Linux 4.4.0-34-generic #53-Ubuntu SMP Wed Jul 27 16:06:39 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

2. Versions. Please run `ng --version`. If there's nothing outputted, please run
in a Terminal: `node --version` and paste the result here:
angular-cli: local (v1.0.0-beta.11-webpack.2, branch: master)
node: 6.4.0
os: linux x64

    `git log` on local angular-cli clone:
    latest master commit is:
    commit 1f4c6fe3e5ac6812f1c40850a086f54b7b698ae3
    Author: Mike Brocchi <mbrocchi@gmail.com>
    Date:   Mon Aug 29 00:05:32 2016 -0400

    feat(feature): add ability to generate feature modules (#1867)

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
added primeng to a project:
npm install --save primeng primeui
added ```"../node_modules/primeui/themes/omega/theme.css",
        "../node_modules/font-awesome/css/font-awesome.min.css",
        "../node_modules/primeui/primeui-ng-all.min.css"``` to apps[0].styles
ng build

4. The log given by the failure. Normally this include a stack trace and some
   more information.
```
ERROR in ./~/primeui/images/loadingbar.gif
Module parse failed: <PROJECT_PATH>/node_modules/primeui/images/loadingbar.gif Unexpected character '�' (1:6)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ <CLI_CLONE_PATH>/angular-cli/~/css-loader!<CLI_CLONE_PATH>/angular-cli/~/postcss-loader!./~/primeui/primeui-ng-all.min.css 6:16675-16709
 @ ./~/primeui/primeui-ng-all.min.css
 @ multi styles

ERROR in ./~/primeui/images/loading.gif
Module parse failed: <PROJECT_PATH>/node_modules/primeui/images/loading.gif Unexpected character '' (1:7)
You may need an appropriate loader to handle this file type.
(Source code omitted for this binary file)
 @ <CLI_CLONE_PATH>/angular-cli/~/css-loader!<CLI_CLONE_PATH>/angular-cli/~/postcss-loader!./~/primeui/primeui-ng-all.min.css 6:32505-32536
 @ ./~/primeui/primeui-ng-all.min.css
 @ multi styles
```

Just ask if any more information is required!

### PR DESCRIPTION
**Pull request by FallenRiteMonk**
add loader to support loading of gif images

Closes #1878

### CODE REVIEW

**Comment by billdwhite** [comment]
+1 on getting this merged in!

**Comment by eabehrens** [comment]
+1 for this!

**Comment by EricBou** [comment]
+1 for this

**Comment by dballance** [comment]
Lack of `.gif` support is a blocker for me  -- looks like a quick win to include this.

**Comment by RicardoVaranda** [comment]
What exactly does this enable you to do?

**Comment by TheLarkInn** [comment]
@FallenRiteMonk 2 things: 

1. What is the purpose for choosing this over url-loader or file-loader?
2. To merge, we need tests verifying this functionality works.

File: addon/ng2/models/webpack-build-common.ts [hunk 1]
```diff
@@ -115,6 +115,12 @@ export function getWebpackCommonConfig(
 
         { test: /\.json$/, loader: 'json-loader' },
         { test: /\.(jpg|png)$/, loader: 'url-loader?limit=10000' },
+        { test: /\.gif$/i,
+          loaders: [
+            'file?hash=sha512&digest=hex&name=[hash].[ext]',
+            'image-webpack?bypassOnDebug&optimizationLevel=7&interlaced=false'
```
Comment (TheLarkInn) [inline addon/ng2/models/webpack-build-common.ts]: Are all these query options for the image min library?

**Comment by FallenRiteMonk** [comment]
1) I don't know. I'm totally new to webpack so I just googled and found this example that worked for me, but I now figured out that url-loader also works and fixed it, hope this is better.
2) Could you please give me a quick heads up on what kind of test would be needed and where it should go? (where are other tests for webpack loader configurations)

**Comment by FallenRiteMonk** [comment]
messed this up with rebase! created a new PR #2066

**Pull request closed without merging**